<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MYO EMG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Home</string>
           </attribute>
           <attribute name="toolTip">
            <string>Related To EMG</string>
           </attribute>
           <widget class="QFrame" name="frame_5">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>-10</y>
              <width>941</width>
              <height>391</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>EMG 1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PlotWidget" name="EMG1"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>EMG 5</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PlotWidget" name="EMG5"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>EMG 2</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PlotWidget" name="EMG2"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>EMG 6</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PlotWidget" name="EMG6"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>EMG 3</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PlotWidget" name="EMG3"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>EMG 7</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PlotWidget" name="EMG7"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>EMG 4</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PlotWidget" name="EMG4"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>EMG 8</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PlotWidget" name="EMG8"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="pushButton_11">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>390</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Start Plot</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_12">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>390</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Stop Plot</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>EMG_Plot</string>
           </attribute>
           <widget class="PlotWidget" name="emgplot">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>811</width>
              <height>421</height>
             </rect>
            </property>
            <property name="accessibleName">
             <string>EMG Plot</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_7">
            <property name="geometry">
             <rect>
              <x>830</x>
              <y>160</y>
              <width>101</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Start Plot</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_8">
            <property name="geometry">
             <rect>
              <x>830</x>
              <y>230</y>
              <width>101</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Stop Plot</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Training</string>
           </attribute>
           <widget class="QPushButton" name="pushButton_9">
            <property name="geometry">
             <rect>
              <x>690</x>
              <y>290</y>
              <width>101</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Save (.csv)</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit">
            <property name="geometry">
             <rect>
              <x>520</x>
              <y>290</y>
              <width>113</width>
              <height>33</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>156</x>
              <y>290</y>
              <width>321</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Number of Records you want to save in .cvs</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_10">
            <property name="geometry">
             <rect>
              <x>690</x>
              <y>220</y>
              <width>101</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Save (.pickle)</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Testing</string>
           </attribute>
           <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
             <rect>
              <x>580</x>
              <y>180</y>
              <width>191</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Start Prediction</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_3">
            <property name="geometry">
             <rect>
              <x>580</x>
              <y>230</y>
              <width>191</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Stop Prediction</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>3000</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Connect MYO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Disconnect MYO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Power OFF MYO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>Clear Console</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1220</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLolo"/>
    <addaction name="actionZozo"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLolo">
   <property name="text">
    <string>Lolo</string>
   </property>
  </action>
  <action name="actionZozo">
   <property name="text">
    <string>Zozo</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
